01week/rockPaperScissors.js

```diff
@@ -13,22 +13,22 @@ function rockPaperScissors(hand1, hand2) {
   let p2 = hand2.toLowerCase().trim()
 
    if (p1 === p2) {
-     return 'It is a Tie!';
+     return "It's a tie!";
    }
   if (p1 === 'rock'){
     if (p2 === 'paper') {
-      return 'Hand 2 Wins!';
-    } else return 'Hand 1 Wins';
+      return 'Hand two wins!';
+    } else return 'Hand one wins!';
   }
   if (p1 === 'scissors') {
     if (p2 === 'paper') {
-      return 'Hand 1 Wins';
-    } else return 'Hand 2 Wins';
+      return 'Hand one wins!';
+    } else return 'Hand two wins!';
   }
   if (p1 === 'paper') {
     if (p2 === 'scissors') {
-      return 'Hand 2 Wins'; 
-    } else return 'Hand 1 Wins';
+      return 'Hand two wins!'; 
+    } else return 'Hand one wins!';
```

Comment:
There is a bit of a fault in the else return logic. is it true, if player 1 doesn't play "paper" and player 2 doesn't play scissors, that player one automatically wins? what if player 1 plays "rock" and player 2 plays "paper"?